pkgs/by-name/op/opcua-commander/package.nix

```diff
@@ -25,10 +26,23 @@ buildNpmPackage rec {
     makeWrapper
   ];
 
-  postPatch = ''
-    substituteInPlace package.json \
-      --replace-warn "npx -y esbuild" "esbuild"
-  '';
+  postPatch =
+    let
+      esbuildPrefix = lib.concatStringsSep " " (
+        [
+          "esbuild"
+        ]
+        ++ lib.optionals stdenv.hostPlatform.isDarwin [
+          # Workaround for 'No loader is configured for ".node" files: node_modules/fsevents/fsevents.node'
+          # esbuild issue is https://github.com/evanw/esbuild/issues/1051
+          "--external:fsevents"
+        ]
+      );
```

Comment:
```suggestion
        esbuildPrefix = "esbuild"
          # Workaround for 'No loader is configured for ".node" files: node_modules/fsevents/fsevents.node'
          # esbuild issue is https://github.com/evanw/esbuild/issues/1051
        ++ lib.optionalString stdenv.hostPlatform.isDarwin "--external:fsevents";
```

Reply:
It was the point where I lacked confidence in determining which function was better for this case.

If I understand that correctly, your suggestion requires some tweaks.

- `++` -> `+`
- Adding a space for arguments separator
- Indentation

So, I will commit the revised diff.